test_unstructured/chunking/test_base.py

```diff
@@ -469,6 +519,50 @@ def it_includes_its_overlap_prefix_in_its_text_when_present(
         )
         assert pre_chunk._text == expected_value
 
+    def it_computes_metadata_for_each_chunk_to_help(self):
+        table = Table("Lorem ipsum", metadata=ElementMetadata(text_as_html="<table/>"))
+        pre_chunk = TablePreChunk(table, overlap_prefix="", opts=ChunkingOptions())
+
+        metadata = pre_chunk._metadata
+
+        assert metadata.text_as_html == "<table/>"
+        # -- opts.include_orig_elements is True by default --
+        assert metadata.orig_elements == [table]
+        # -- it produces a new instance each time it is called so changing one chunk's metadata does
```

Comment:
Short answer, yes. The docstring on the `TablePreChunk._metadata` property give more, but the gist is every `.metadata` object for a chunk produced from a `TablePreChunk` will have at least one unique value, for `.text_as_html` for one, `.is_continuation` sometimes another.

So when we form the "base" metadata it always needs to be a new object because it can be mutated in `TablePreChunk.iter_chunks()`. Note that `._metadata` is an `@property` and not an `@lazyproperty`, so it gets computed on each reference.